buildkite/src/Jobs/ArchiveNode.dhall

```diff
@@ -41,7 +41,7 @@ Pipeline.build
                 (Prelude.Text.concatSep " && "
                   [ "bash buildkite/scripts/setup-database-for-archive-node.sh ${user} ${password} ${db}"
                   , "PGPASSWORD=${password} psql -h localhost -p 5432 -U ${user} -d ${db} -a -f src/app/archive/create_schema.sql"
-                  , WithCargo.withCargo "dune runtest src/app/archive"
+                  , WithCargo.withCargo "eval $(opam config env) && dune runtest src/app/archive"
```

Comment:
I would be surprised if this works for you, check the logs to make sure the opam config init wasn’t evaluated too early. I can never get escaping to work properly for these so I always fall back to moving it to a shell script

Reply:
It appears to work, archive node tests no longer fail at line 460 like: https://buildkite.com/o-1-labs-2/coda/builds/870#fafecf47-0127-4f63-a9f9-ba4d9221b782